### DISCUSSION TITLE
Starting a team on CKM discussion

### DISCUSSION THREAD

**Discussion by ppdewolf** [sdcTools/UserSupport#241] [CKM Discussion]
Only waiting for Reinhard Tent: I coudln't find him at github, so probably he still has to make a github account?

**Comment by tenderle** [comment]
Github still warns that our browsers are out of date. Fortunately, Firefox meanwhile allows for the "Comment"-functionality. Half a year ago it was not possible.

**Comment by Sarah-Giessing** [comment]
For me too it works now, though only after switching from the default Internet Explorer to Firefox

**Comment by bernhard-da** [comment]
great, works well!

**Comment by r-tent** [comment]
Hi there! Sorry for being late. Yet I did have an account ;-)

**Comment by ppdewolf** [comment]
@r-tent That what I thought, but I couldn't find it that quickly.

**Comment by tenderle** [comment]
Dear @ppdewolf could you invite our colleague Felix Geyer (@geyerf) to the SDCtools organization? He may be interested in this discussion.

**Comment by ppdewolf** [comment]
@tenderle Okay. Just invited him.

**Comment by ppdewolf** [comment]
By the way: the dicsussion is visible to everyone, also non-team-members. That's not a problem I guess?

**Comment by Sarah-Giessing** [comment]
Would there be an easy way for making it invisible outside the team? Of course our final choices will be visible in the (open source). But we might sometimes discuss preferences which we wouldn't discuss in public.

**Comment by tenderle** [comment]
@ppdewolf Is it visible to everyone on github or only for sdcTools Members?

**Comment by tenderle** [comment]
IIRC @geyerf wasn't able to see anything before he was invited to the organization

**Comment by bernhard-da** [comment]
should be visible only to invited members

**Comment by ppdewolf** [comment]
Sorry, I meant that it is visible to everyone in the organization sdcTools. Not everyone on github.
